docker/Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 		pkg-config \
 	&& rm -rf /var/lib/apt/lists/*
 
-ENV GOLANG_VERSION 1.10.3
+ENV GOLANG_VERSION 1.11
```

Comment:
Does that work? Below it will also check the md5 hash which has changed. Have a look at the other dockerfile change: https://github.com/gaia-pipeline/gaia/pull/146/files#diff-192af0c41f653dbf5bf864ba3ac4c682

Reply:
Uh, nice. I wonder if that's the problem. :D

Reply:
Oh right, I totally thought it is by the changes of the hashes in the above. I'll fix these. :)